docs/eog/case-studies-cw20.rst

```diff
@@ -0,0 +1,42 @@
+.. _CW20:
+
+.. Comment - `Link text <https://domain.invalid/>`_
+
+Collaborations Workshop 2020 (CW20) - Online 
+=============================================
+
+The Collaborations Workshop (CW) is the annual flagship event of the Software Sustainability Institute. It brings together researchers, developers, innovators, managers, funders, publishers, leaders, and educators to explore and share best practice.
+
+On 5 March - just over three weeks before Collaborations Workshop 2020 (CW20) was due to start in Belfast, Northern Ireland - everything was in place. The programme had been finalised, keynote speakers announced, catering booked and accommodation assigned. There had been some issues - a group from Northern Italy had cancelled because they had been quarantined due to an outbreak of a novel coronavirus. At this point the number of reported cases in the UK was small, but things were moving rapidly. The clear message was that we couldn’t run the event as planned. 
```

Comment:
```suggestion
On 5 March 2020 - just over three weeks before Collaborations Workshop 2020 (CW20) was due to start in Belfast, Northern Ireland - everything was in place. The programme had been finalised, keynote speakers announced, catering booked and accommodation assigned. There had been some issues - a group from Northern Italy had cancelled because they had been quarantined due to an outbreak of a novel coronavirus. At this point the number of reported cases in the UK was small, but things were moving rapidly. The clear message was that we couldn’t run the event as planned. 
```

Reply:
Aside: I just noticed the agenda - https://software.ac.uk/cw20/agenda should have a link to the speed blogs produced - perhaps using the tags - e.g. https://software.ac.uk/tags/cw20-speed-blog-posts